frameworks/FSharp/giraffe/benchmark_config.json

```diff
@@ -37,27 +38,28 @@
         "webserver": "Kestrel",
         "os": "Linux",
         "database_os": "Linux",
-        "display_name": "Giraffe",
+        "display_name": "Giraffe, utf8json",
         "notes": "",
         "versus": "aspcore"
       }
     },
     {
-      "utf8plaintext": {
+      "stripped": {
+        "fortune_url": "/fortunes",
         "plaintext_url": "/plaintext",
         "port": 8080,
-        "approach": "Realistic",
-        "classification": "fullstack",
-        "database": "None",
+        "approach": "Stripped",
```

Comment:
Its still `Realistic` and likely to become the defacto method once `Utf8String` becomes mainline https://github.com/dotnet/corefxlab/issues/2350

Reply:
In recent run its 1,347,868.6 rps vs 1,220,305.2 rps for `giraffe-utf8plaintext` vs `giraffe`